src/DirectAdmin/DirectAdmin.php

```diff
@@ -101,8 +101,6 @@ protected function __construct($url, $username, $password)
         $this->connection = new Client([
             'base_uri' => $this->baseUrl,
             'auth' => [$username, $password],
-            'http_errors' => true,
```

Comment:
Hmm you're right, in that case I must be wrong about the return codes hehe. Also the CI runs the test on a live server after merge so we'll see if it passes  ;)